pkgs/applications/audio/sunvox/default.nix

```
@@ -13,11 +13,14 @@ let
 in
 stdenv.mkDerivation rec {
   pname = "SunVox";
-  version = "2.1c";
+  version = "2.1.1c";
 
   src = fetchurl {
-    url = "https://www.warmplace.ru/soft/sunvox/sunvox-${version}.zip";
-    sha256 = "sha256-yPVcbtlAVbO9uMsFlfZ51T408hA1VPJAI+R+Jdjcyjw=";
+    urls = [
+      "https://www.warmplace.ru/soft/sunvox/sunvox-${version}.zip"
+      "https://web.archive.org/web/20231204012052/https://www.warmplace.ru/soft/sunvox/sunvox-2.1.1c.zip"
```

```suggestion
      "https://web.archive.org/web/20231204012052/https://www.warmplace.ru/soft/sunvox/sunvox-${version}.zip"
```
we shouldn't forget to bump this otherwise we could run into strange unreproducible bugs.

---

The link has a date of when the target URL was archived, just substituting the version part won't work.

---

That also true.. I am not sure. What about just using the web.archive.org link? Would that be better?

---

The upstream download is noticeably faster, and going archive-only would still require updating the link every time.

Since the archive link is sandwiched between `version` and `src.sha256`, it should be visible in diffs. And the hardcoded version in the URL should stick out to reviewers when it hasn't been updated. We could also add a comment above the version, to remind ppl that the archive url needs to be updated.

---

I have added a comment above the archive.org mirror, as a reminder that this `src` must be backed up. Is this better?

---

Since I also do not have a better idea, this must be good enough